### ISSUE TITLE
Successor to MDL is MDC

### ISSUE THREAD

**Issue by ghost** [MikeMitterer/dart-material-design-lite#46]
Wondering if you are thinking of upgrading ? Maybe there is a dart implementation of MDC.
MDL is end of life.

**Comment by MikeMitterer** [comment]
Yup! I'll upgrade. But first I throw out all dependencies to mirrors and switch over to either reflectable or to build_mirrors.

**Comment by latesr** [comment]
Hi Mike.  I'm investigating using dart with MDC for web (to work alongside a flutter app) and was wondering how you're getting on with the migration to MDC.

**Comment by ghost** [comment]
@latesr thats exactly what i am also looking to do :)
